README.md

```diff
@@ -310,7 +315,7 @@ Determinator::Tracking.get_context do
 end
 ```
 
-NOTE: this is implemented by keeping the list of requests in a per-request thread-local variable, which means that determinations will only be tracked on the main thread.
+NOTE: tracking is implemented by keeping the list of requests in a per-request thread-local variable, which means that determinations will only be tracked on the main thread.
```

Comment:
This is kind of confusing - earlier in the README it says that tracking is done using thread-local variables, so I'd expect tracking to work in any thread provided that thread was wrapped up in the requisite middleware. Sidekiq for example can use multiple worker threads when processing jobs.

Reply:
Maybe the note is not as clear as it could be. This will work with multiple worker threads, but will not track determinations if the determination is made from a thread that is spun from a worker thread.

```
# will not work

1.upto(10) { Thread.new { do_work if Determinator.instance.feature_flag_on?(:my_experiment_name) } }

# will work

on = Determinator.instance.feature_flag_on?(:my_experiment_name)
1.upto(10) { Thread.new { do_work if on } }
```

The following paragraph was supposed to make it clearer, do you think the wording can be improved?

Reply:
Taken a swing at this:

> N.B. determinations will only be recorded on threads that initialise `Determinator::Tracking` via middleware. Please take care when offloading work away from these threads, as their determinations will not be tracked.

I've tried to avoid using the term "main thread", and instead explicitly mentioned that only threads that use the middleware (i.e. the "main threads" in puma/sidekiq contexts). I'm not too happy with "these threads" as a substitute for "application threads" or "main threads" — maybe that can be refined further?

Reply:
Tried rewording, please see if it makes more sense!